code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -78,6 +82,9 @@
 
 	allowed_module_types = MODULE_GENERAL | MODULE_UTILITY
 
+/obj/item/clothing/suit/space/rig/industrial
+	flags_inv = HIDETAIL
```

Comment:
Why this does not have HIDEJUMPSUIT?

Reply:
With how long the boots and gauntlets are, the suit appears silly when you retract them without a jumpsuit being beneath it.
The suit covers most jumpsuits anyhow, with only small, non-standard edge cases having the potential of being a problem.